### ISSUE TITLE
How to use self-built datasets for training

### ISSUE THREAD

**Issue by yeuyuetou** [isl-org/ZoeDepth#48]
For the training part in the readme, I don't understand if this is fine-tuning with NYU-Depth V2, or retraining, but not using 10 datasets for relative depth training and 2 datasets for metric depth training, sorry, I'm a novice, please forgive me if the problem is a bit stupid

**Comment by yeuyuetou** [comment]
Hello!  I use python train_mono.py -m zoedepth --pretrained_resource="local::C:Users20563.cachetorchhubcheckpointsdpt_beit_large_384.pt"
Training, but stuck in Using pretrained resource local::C:Users20563.cachetorchhubcheckpointsdpt_beit_large_384.pt
No more moving, and GPU usage is only 2 percent (my device is not very good)

**Comment by Shiyao-Xu** [comment]
Hi! Have you ever met with a problem when training, that the model weights mismatch the model definition?

**Comment by Jkdxbns** [comment]
Hello! I am totally new to AIML and I need to use ZoeDepth library for depth estimation. Since I am working on microscopic imaging, I need to train the model on my dataset. I suppose that can be done using fine tuning, but I don't know from where to start or which resources to seek help from, can anyone help me with it?
